components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -180,6 +183,12 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
         if uuid_is_blacklisted(self.uuid.as_ref(), Blacklist::Writes) {
             return Err(Security)
         }
+        if value.len() > MAXIMUM_ATTRIBUTE_LENGTH {
+            return Err(NoModificationAllowed)
+        }
+        if !self.Service().Device().Gatt().Connected() {
```

Comment:
I suppose `Service()`, `Device()` and `Gatt()` always return a valid value, right?